src/main/java/technology/tabula/extractors/BatchSelectionExtractor.java

```diff
@@ -0,0 +1,473 @@
+package technology.tabula.extractors;
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.commons.cli.ParseException;

```

Comment:
`org.apache.commons.cli.ParseException` is only used for indicating syntax errors when parsing command line arguments.